spec/controllers/group_assignment_invitations_controller_spec.rb

```diff
@@ -22,7 +22,7 @@
                              title: 'HTML5',
                              grouping: grouping,
                              organization: organization,
-                             public_repo: false)
+                             public_repo: true)
```

Comment:
It was because I was using my personal token and I didn't want the creation to show up on my timeline.

No biggie